src/ui/components/results/resultsheadercomponent.js

```diff
@@ -12,6 +12,7 @@ const DEFAULT_CONFIG = {
   resultsCountSeparator: '|',
   verticalURL: undefined,
   showChangeFilters: false,
+  changeFiltersText: 'change filters',
```

Comment:
It seems that the default for the hidden, top-level changeFiltersText was just 'change'. Do we need to use that default here?

Reply:
we can keep the old accordion results behavior. the new default text for should be 'change filters', though

Reply:
another breaking change, I'll ask Rose if its okay